packages/notifications/src/styled/global-alert/StyledGlobalAlertTitle.ts

```diff
@@ -43,7 +43,7 @@ export const StyledGlobalAlertTitle = styled.div.attrs({
   'data-garden-id': COMPONENT_ID,
   'data-garden-version': PACKAGE_VERSION
 })<IStyledGlobalAlertTitleProps>`
-  display: inline;
+  display: inline-flex;
```

Comment:
Unfortunately, I don't think this solution scales as expected for a long title. With `inline-flex` we see the following:

<img width="608" alt="Screen Shot 2022-12-15 at 9 03 16 AM" src="https://user-images.githubusercontent.com/143773/207909236-b3986ba0-5c02-461c-9c1e-f1372d386418.png">

But design intent is for content to flow like this:

<img width="606" alt="Screen Shot 2022-12-15 at 9 05 36 AM" src="https://user-images.githubusercontent.com/143773/207909636-88a74387-dd62-4d74-9436-7505b3fd3ebe.png">

Reply:
Updated, good catch. Hooked up `direction` to `theme.rtl` :+1: